scripts/mantidpython.py

```diff
@@ -3,45 +3,156 @@
 import re
 import subprocess
 import sys
+import tempfile
 
-mantidRegExp=re.compile(r'/opt/.antid.*/bin')
-def getMantidLoc(line):
+mantidRegExp = re.compile(r'/opt/.antid.*/bin')
+
+MANDID_VERSION = 'MANTID_VERSION='
+CONDA_NAME = 'CONDA_ENV='
+NSD_CONDA_WRAP = '/SNS/users/wzz/Projects/AutoReduction/nsd-app-wrap/nsd-conda-wrap.sh'
```

Comment:
Should refactor this to search in some locations for the script (like a PATH where the first one wins: `/usr/bin` then `/SNS/software/bin`) rather than hard-code the location.

Reply:
@peterfpeterson Have you deployed the new `nsd-conda-wrap.sh`?